cmd/mattermost/commands/user.go

```diff
@@ -368,6 +379,52 @@ func userCreateCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func usersToBots(command *cobra.Command, args []string, a *app.App) error {
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	users := getUsersFromUserArgs(a, args)
+	for i, user := range users {
+		if user == nil {
+			CommandPrintErrorln(fmt.Errorf("Unable to find user \"%s\"", args[i]))
+			continue
+		}
+
+		bot, err := a.ConvertUserToBot(user)
+		if err != nil {
+			CommandPrintErrorln(err.Error())
+			continue
+		}
+
+		CommandPrettyPrintln("ownerId: " + bot.OwnerId)
+		CommandPrettyPrintln("userId: " + bot.UserId)
+		CommandPrettyPrintln("username: " + bot.Username)
+		CommandPrettyPrintln("displayName: " + bot.DisplayName)
+	}
+
+	return nil
+}
+
+func userModifyCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	toBot, err := command.Flags().GetBool("bot")
+	if err != nil {
+		return errors.New("Invalid command. See help text for details.")
+	}
+
+	if toBot {
```

Comment:
The idiomatic way to do this would be handle the `!toBot` and return the error, and the happy path at first level.

Reply:
Another option is to mark the `--bot` flag as required in the command. (see here: https://github.com/spf13/cobra#required-flags)

Reply:
I am having a follow up to implement the portion where `toBot` is false. But yeah, for now I can use the idiomatic way.